tests/Doctrine/EntityRegeneratorTest.php

```diff
@@ -113,7 +113,7 @@ private function doTestRegeneration(string $sourceDir, Kernel $kernel, string $n
             });
 
         $fileManager = new FileManager($fs, $autoloaderUtil, new MakerFileLinkFormatter(null), $tmpDir);
-        $doctrineHelper = new DoctrineHelper('App\\Entity', $container->get('doctrine'));
+        $doctrineHelper = new DoctrineHelper('App\\Entity', $container->get('doctrine'), false, null);
```

Comment:
revert, `false` and `null` are the defaults values

Reply:
done